plugins/wepselect.lua

```diff
@@ -153,7 +163,11 @@ if (CLIENT) then
 			end
 
 			return true
-		elseif (bind:find("invnext") and !bTool) then
+		elseif (bScrollDown) then
+			if (bHoldingToolgun and tool and tool.Scroll and tool:Scroll(tr, 0)) then
```

Comment:
Duplicate with L154, can pull out of the scroll direction conditional?

Reply:
if bHoldingToolgun block is only hit once, if bScrollDown or bScrollUp is true
if it's not inside the scroll conditionals, it'll constantly be scrolling

Reply:
Yeah that makes sense, would be quite a long single conditional